style/icomoon/demo.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
```

Comment:
I'm not sure, but I don't think it get's bundled since I only send dist to production, and index.html so demo.html doesn't leave local pc